glideinmonitor/webserver/static/condor_jobview.js

```diff
@@ -0,0 +1,217 @@
+/*
+* Helper functions
+*/
+function var_win(data, name) {
+    // Open a variable's contents in a new window
+    let newWindow = window.open("", name, "width=600, height=800");
+    newWindow.document.write("<pre>" + data + "</pre>");
+    newWindow.document.title = name;
+}
+
+
+function var_download(text, filename) {
+    // Variable downloader
+    let blob = new Blob([text], {type: "text/plain;charset=utf-8"});
+    saveAs(blob, filename);
+}
+
+
+function parseTarString(text) {
+    // Parses a inflated gzip file that's still in a tar? format
+    // Returns a 2D array containing the name and text of each file
+    let curr_file = -1;
+    let content = [];
+
+    const lines = text.split('\n');
+    for (let i = 0; i < lines.length; i++) {
+        // For each line, check if the line is a TAR file header
+        if (lines[i].includes(String.fromCharCode(0)) && lines[i].includes("ustar")) {
+            // Increment the index of the content variable
+            curr_file += 1;
+            content[curr_file] = ["", ""];
+
+            // Get the file name from the header
+            let charStart = 0;
+            while (lines[i].charCodeAt(charStart) === 0) {
```

Comment:
I think this should be refactored together w/ jobview.js int 3 files to avoid repeating some functions in multiple files.

Reply:
I agree, should I open a new ticket for this refactoring?